npcs/generalNpc.ts

```diff
@@ -87,6 +87,25 @@ export default class GeneralNpc extends Trigger {
         }
     }
 
+    public handlePlayerImageCollision(playerImage: Phaser.Physics.Arcade.Sprite, collisionImage: Phaser.Physics.Arcade.Sprite){
```

Comment:
Method was declared as protected in trigger.ts, now becomes public. I think it was meant to stay protected.